### PR TITLE
Context module update from master

### PR DESCRIPTION
**Pull request by spiderbiggen**
This should remove code duplication between the context environment and the tygron environment also includes a version bump from master 1.0.10 -> 1.0.11
bumps contextvh environment version to 1.0.1

### CODE REVIEW

**Comment by spiderbiggen** [comment]
travis build log https://travis-ci.org/spiderbiggen/tygron/builds/135890898

**Comment by klikooo** [comment]
Tested it with the agent and it works correctly.

**Comment by nskartoredjo** [comment]
Looks great, I/we will test it with our agent and if it passes than we aprove

**Comment by nskartoredjo** [comment]
Tested with our agent, it works correctly. We approve. -CTD

File: contextvh/pom.xml [hunk 1]
```diff
@@ -47,19 +48,21 @@
...
-            <groupId>nl.tygron</groupId>
-            <artifactId>sdk</artifactId>
-            <version>2016.5.0.5</version>
-        </dependency>
-        <dependency>
             <groupId>eishub</groupId>
             <artifactId>eis</artifactId>
             <version>0.5.0</version>
         </dependency>
 		<dependency>
```
Comment (dionytadema) [inline contextvh/pom.xml]: indentation is off

File: contextvh/pom.xml [hunk 2]
```diff
@@ -47,19 +48,21 @@
...
             <groupId>eishub</groupId>
             <artifactId>tygronenv</artifactId>
-            <version>1.0.10</version>
+            <version>${tygronenv.version}</version>
```
Comment (dionytadema) [inline contextvh/pom.xml:58]: add version number

File: contextvh/pom.xml [hunk 3]
```diff
@@ -47,19 +48,21 @@
...
+        </dependency>
+        <dependency>
+            <groupId>eishub</groupId>
+            <artifactId>tygronenv</artifactId>
+            <version>${tygronenv.version}</version>
```
Comment (dionytadema) [inline contextvh/pom.xml:63]: add version number

File: contextvh/src/main/java/contextvh/translators/ContextJ2Stakeholder.java [hunk 4]
```diff
@@ -40,10 +40,10 @@
                         new Numeral(budget),
                         new Numeral(stakeholder.getYearlyIncome())
                 ), new Function(
-                        "indicatorLink",
-                        new Numeral(stakeholder.getID()),
-                        indicator(stakeholder.getMyIndicators(), stakeholder)
-                )
+                "indicatorLink",
```
Comment (dionytadema) [inline contextvh/src/main/java/contextvh/translators/ContextJ2Stakeholder.java]: why did the indentation change here, it was correct

**Comment by dionytadema** [comment]
also please don't include multiple changes ( eishub update to 1.0.11 and removal of code duplication) in a single PR as it now has so many changes that it is hard to review

**Comment by klikooo** [comment]
@dionytadema these are updates from wouter and frank so that explains why it is a lot...

**Comment by dionytadema** [comment]
@klikooo i get that, and it's fine for now, but it would have been nicer if the changes of frank would be in a separate PR, now it's hard to see what was changed by you and what is part of the update to 1.0.11

**Comment by spiderbiggen** [comment]
@dionytadema It's not that hard to see I only changed files in the contextvh folder/module. And apart from that most of the changes in the master were explicitly made to reduce the amount of code duplication between the two modules

**Comment by MatthijsJWH** [comment]
We also tested it with our agents and it works fine.
We approve of the changes.
- CodeFox

File: contextvh/src/main/java/contextvh/translators/ContextJ2Stakeholder.java (hunk 4 above)
Comment (spiderbiggen) [inline contextvh/src/main/java/contextvh/translators/ContextJ2Stakeholder.java]: this is a good question. I don't know why

File: contextvh/pom.xml (hunk 2 above)
Comment (spiderbiggen) [inline contextvh/pom.xml:58]: What do you mean?

**Comment by jzoon** [comment]
Well done Stefan, I don't think it's necessary to split the pullrequest to have Franks changes in a seperate one.

**Comment by Denpeer** [comment]
WhySoSerious approves

**Pull request merged**